services/121-service/test/payment/payment-create-and-start.test.ts

```diff
@@ -131,65 +133,162 @@ describe('Start and create a payment separately', () => {
     expect(registrationAfterStart!.paymentCount).toBe(1);
   });
 
-  it('should not start transactions of non-included registrations', async () => {
+  describe('process included registrations only on payment starts', () => {
+    let paymentId: number;
```

Comment:
This should not be a `let` here. 

I think you need to create a new `const paymentId` after every create payment